gio/Gir.toml

```diff
@@ -718,6 +717,66 @@ status = "generate"
     #unsupported memory allocators
     ignore = true
 
+[[object]]
+name = "Gio.MenuAttributeIter"
+status = "generate"
+    [[object.function]]
+    name = "get_name"
+    # Users shoud use `next` (automatically renamed from `get_next`).
+    ignore= true
+    [[object.function]]
+    name = "get_value"
+    # Users shoud use `next` (automatically renamed from `get_next`).
+    ignore= true
+    [[object.function]]
+    name = "next"
+    # this one advances the iter and returns a boolean
+    # `get_next` (which will be automatically renamed as `next`)
+    # matches what we expect from an iterator in Rust.
+    ignore= true
+
+[[object]]
+name = "Gio.MenuLinkIter"
+status = "generate"
+    [[object.function]]
+    name = "get_name"
+    # Users shoud use `next` (automatically renamed from `get_next`).
+    ignore= true
+    [[object.function]]
+    name = "get_value"
+    # Users shoud use `next` (automatically renamed from `get_next`).
+    ignore= true
+    [[object.function]]
+    name = "next"
+    # this one advances the iter and returns a boolean
+    # `get_next` (which will be automatically renamed as `next`)
+    # matches what we expect from an iterator in Rust.
+    ignore= true
+
+[[object]]
+name = "Gio.Mount"
+status = "generate"
+    [[object.function]]
+    name = "get_default_location"
+        [object.function.return]
+        nullable = false
```

Comment:
Why all the `nullable = false` here?

Reply:
I can't tell now... and it doesn't seem to make any difference. Removing.